src/ui/popup/manage-settings-page/export-theme.tsx

```diff
@@ -0,0 +1,34 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import {Button} from '../../controls';
+import {saveFile} from '../../utils';
+import ControlGroup from '../control-group';
+
+export default function ExportTheme(props: ViewProps) {
+    const listener = ({type, data}) => {
+        if (type === 'export-css-response') {
+            saveFile('Dark-Reader-Settings.css', data);
+            chrome.runtime.onMessage.removeListener(listener);
+        }
+    };
+
+    function exportCSS() {
+        chrome.runtime.onMessage.addListener(listener);
+        chrome.runtime.sendMessage({type: 'export-css-proxy', data: {url: props.tab.url}});
```

Comment:
Do you need the URL here? Since active tab is available in `props.tab` you can use `tab.id`, no need in `getActiveTab`.